src/Pages/Home.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { spinnerService } from "@simply007org/react-spinners";
+import { useEffect, useState } from "react";
+import { Client } from "../Client";
+import Banner from "../Components/Banner";
+import LatestArticles from "../Components/LatestArticles";
+import LinkButton from "../Components/LinkButton";
+import Metadata from "../Components/Metadata";
+import OurStory from "../Components/OurStory";
+import TasteOurCoffee from "../Components/TasteOurCoffee";
+import { getAboutUsLink } from "../Utilities/ContentLinks";
+import { defaultLanguage, initLanguageCodeObject } from "../Utilities/LanguageCodes";
+import { useIntl } from 'react-intl';
+import { Home as HomeType } from '../Models/home';
+import { projectModel } from '../Models/_project';
+
+interface HomeProps {
+}
+
+const Home: React.FC<HomeProps> = () => {
+  const { locale:language, formatMessage } = useIntl();
+  const [homeData, setHomeData] = useState(initLanguageCodeObject<HomeType>());
+
+  useEffect(() => {
+
+    spinnerService.show("apiSpinner");
+
+    const query = Client.items<HomeType>()
+      .type(projectModel.contentTypes.home.codename);
+    if (language) {
+      query.languageParameter(language);
+    }
+
+    query
+      .toPromise()
+      .then(response => {
+
+        const currentLanguage = language || defaultLanguage;
+
+        spinnerService.hide("apiSpinner");
+        setHomeData((data: any|null) => ({
```

Comment:
It is not necessary to specify type it is inteferred by the method signature.

Reply:
[8aa5bf7](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/8aa5bf731ee5707d143c64d9e447f07bd20b49e2)